spread.yaml

```diff
@@ -143,6 +143,9 @@ backends:
             - ubuntu-23.04-64:
                   storage: 12G
                   workers: 8
+            - ubuntu-23.10-64:
+                  storage: 12G
+                  workers: 8
```

Comment:
Let's please also add it to the qemu backends

Reply:
done, thanks for the review